jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/TesterBase.java

```diff
@@ -44,7 +45,7 @@
 public class TesterBase extends BrowserDriverManagingTesterBase {
 
 	// Logger
-	private static final Logger logger = LoggerFactory.getLogger(TesterBase.class);
+	protected static final Logger logger = LoggerFactory.getLogger(TesterBase.class);
```

Comment:
Usually we make loggers `private`. If you need to use a logger in a subclass, create a new one. I believe that makes logs easier to read. @ngriffin7a, may correct me though.